app/Http/Controllers/Maps/DeviceDependencyController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * DependencyController.php
+ *
+ * Controller for graphing Relationships
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2019 Thomas Berberich
+ * @author     Thomas Berberich <sourcehhdoctor@gmail.com>
+ */
+
+namespace App\Http\Controllers\Maps;
+
+use App\Models\Device;
+use Illuminate\Http\Request;
+use LibreNMS\Util\Url;
+
+class DeviceDependencyController extends MapController
+{
+    protected static function deviceList($request) {
+
+        $group_id = $request->get('group') | 0;
```

Comment:
`|` is a bitwise operator, you should use `||` if you want an `or`.   
In this case tho, You could just remove it

Reply:
oh, yes you're right. :-)
i changed it to "||"